susemanager-utils/susemanager-sls/salt-ssh/preflight.sh

```diff
@@ -95,6 +95,9 @@ function getY_CLIENT_CODE_BASE() {
     if [ -L /usr/share/doc/sles_es-release ]; then
         BASE="res"
         VERSION=6
+    elif [ -f /etc/openEuler-release ]; then
```

Comment:
Same here, wasn't the bootstrap already working using `/etc/os-release`? Why do we need this extra code?

Reply:
Salt-ssh still works as well. So unsure whether to keep last requested changes or not.

Reply:
Good, do not remove this. Same comment as above.